include/obmc_console.hpp

```diff
@@ -132,6 +133,67 @@ inline void requestRoutes(App& app)
                     conn.getIoContext());
             hostSocket->async_connect(ep, connectHandler);
         }
+        // Ensure user has ConfigureManager, setting above does nothing
```

Comment:
Any reason we haven't sent upstream? I could see some config option where you could input minimum permission level as making it through.

Reply:
So the user management code is a bit of a mess. 

        .privileges({{"ConfigureComponents", "ConfigureManager"}})

or 
      
        .privileges({{ "ConfigureManager"}})

doesn't actually work. We need a commits like https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/46990/ and 
https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/46991/ but when we pulled that in as part of https://github.com/ibm-openbmc/bmcweb/pull/329, bad things happened (https://github.com/ibm-openbmc/bmcweb/pull/345). 
So this, and checks like https://github.com/ibm-openbmc/bmcweb/blob/1050/include/obmc_hypervisor.hpp#L129 and https://github.com/ibm-openbmc/bmcweb/blob/1050/http/obmc_shell.hpp#L190 are there as a temporary hack. Hopefully this all gets sorted upstream.